.cloudbees/workflows/workflow.yaml

```diff
@@ -11,6 +11,15 @@ jobs:
     steps:
     - name: get source code
       uses: cloudbees-io/checkout@v1
+    - name: Run bats tests
+      uses: docker://bash:5-alpine3.18
+      run: |
+        wget -qO - https://github.com/bats-core/bats-core/archive/refs/tags/v${BATS_VERSION}.tar.gz | tar -C /tmp -xzf -
```

Comment:
`bats` has a docker image: https://hub.docker.com/r/bats/bats
It can simplify this step